src/dspbase.jl

```diff
@@ -677,6 +676,9 @@ function _conv(u, v, su, sv)
     _conv!(out, u, v, su, sv, outsize)
 end
 
+# We use this type definition for clarity
+const AbstractFloats = Union{<:AbstractFloat,Complex{T} where T<:AbstractFloat}
```

Comment:
I'm too happy with the name `AbstractFloats`. Not that I have a good idea for an alternative, though. Maybe `RealOrComplexFloat`?

Reply:
Your name is more descriptive, so I changed it.